androidApp/build.gradle.kts

```diff
@@ -33,8 +33,8 @@ android {
         applicationId = "com.jetbrains.kotlinconf"
         targetSdk = libs.versions.android.targetSdk.get().toInt()
         minSdk = libs.versions.android.minSdk.get().toInt()
-        versionCode = 38
-        versionName = "38.0"
+        versionCode = 62
+        versionName = "25.0.0"
```

Comment:
nope, we can't go back in time

Reply:
This is only the user-visible version string on Android, as far as I know, we can pick any arbitrary value for it. There's nothing checking that it increments over time.

https://developer.android.com/studio/publish/versioning#versioningsettings